src/API/REST/V3/Routes/Donations/DonationNotesController.php

```diff
@@ -381,7 +381,7 @@ public function get_item_schema(): array
                 'type' => [
                     'description' => __('The type of the note.', 'give'),
                     'type' => 'string',
-                    'enum' => ['admin', 'donor'],
+                    'enum' => ['admin', 'donation', 'donor'],
```

Comment:
@JoshuaHungDinh why do we need to update the donation notes controller I though this was for subscription notes?

cc: @pauloiankoski